src/Modal.js

```diff
@@ -33,24 +32,14 @@ import { ModalCard } from './Modal/ModalCard.js'
  * @see Specification: {@link https://github.com/dhis2/design-system/blob/master/molecules/modal.md|Design system}
  * @see Live demo: {@link /demo/?path=/story/modal--small-title-content-action|Storybook}
  */
-export const Modal = ({ children, onClose, small, large, open, className }) => {
+export const Modal = ({ children, onClose, small, large, className }) => {
```

Comment:
Since we're returning directly you could omit the return keyword and use parentheses.